react/Textarea/Textarea.less

```diff
@@ -6,12 +6,12 @@
 }
 
 .textarea {
-  .rawText();
+  .touchableText();
```

Comment:
Is this mixin worth it, given we're overriding the line height? Thinking it might be better to inline the rules, since it's not semantically "touchable text" anyway.